steps/approved-premises/applications/add-exemption-details.ts

```diff
@@ -15,7 +15,7 @@ export const addExemptionDetails = async (page: Page) => {
         .getByLabel('Provide a summary of the reasons why this is an exempt application')
         .fill('Test reason for application exemption')
     await page.locator('.govuk-button', { hasText: 'Save and continue' }).click()
-    await expect(page.locator('.govuk-fieldset__heading')).toHaveText(
-        'Which of the following best describes the sentence type the person is on?'
+    await expect(page.locator('#main-content h1')).toContainText(
+        /.*Is [A-Z][a-z'’-]+([ -][A-Z][A-z'’-]+)* transgender or do they have a transgender history?.*/
     )
```

Comment:
Do you need this regex?  If you're just using it to check you're on the right page, maybe simplify it to something like:
```suggestion
    await expect(page.locator('#main-content h1')).toContainText(/do they have a transgender history?/)
```